portality/view/oaipmh.py

```diff
@@ -157,6 +165,8 @@ def make_resumption_token(metadata_prefix=None, from_date=None, until_date=None,
         d["s"] = oai_set
     if start_number is not None:
```

Comment:
start_number can be removed

Reply:
I'll stop writing that now - throughout there are various places where start_number appears and can just be removed.